src/main/java/com/outbrain/ci/friendly/flatten/maven/plugin/visitor/PomVisitor.java

```diff
@@ -0,0 +1,17 @@
+package com.outbrain.ci.friendly.flatten.maven.plugin.visitor;
+
+public interface PomVisitor {
```

Comment:
Why do you need an interface? there's only one implementation.

Reply:
In the beginning i thought about visitor for each property . But then i aggregate them into one .
Might be will split them again in the near future